test/built-ins/Proxy/get/call-parameters-2.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
For consistency with the other tests for this trap, you can merge this with `get/call-parameters.js`